akamai.services.yml

```diff
@@ -1,4 +1,7 @@
 services:
+  akamai.edgegridclient:
```

Comment:
Not so sure about this name for the service. Might need a follow up issue to change it.